net/gatling/patches/010-decrease_optimisation.patch

```diff
@@ -0,0 +1,13 @@
+Index: gatling-0.16/GNUmakefile
+===================================================================
+--- gatling-0.16.orig/GNUmakefile
++++ gatling-0.16/GNUmakefile
+@@ -47,7 +47,7 @@ ifneq ($(DEBUG),)
+ CFLAGS+=-g -Og
+ LDFLAGS+=-g
+ else
+-CFLAGS+=-O2 -fomit-frame-pointer -I/usr/local/include
++CFLAGS+=-O1 -fomit-frame-pointer -I/usr/local/include
```

Comment:
There is missing commit description, I wonder why you decreased optimisation.

Reply:
If compiling with `-O2`, it will brake on mips-architectures, as the linker can't link the different modules together. It asks for enabling interlinking, which afaict enabled on the target but default. But it wasn't working for me. So decreasing the optimization was a quite easy workaround for me.

Reply:
This seems weird to me. In any case, this needs to be said in the commit description, because no else would know why it was decreased.

Reply:
I added a description in the patch-header now.